src/app/modules/shared_models/token_model.nim

```diff
@@ -200,6 +200,10 @@ QtObject:
         if(balance.balance.getAmount() >= requiredGas):
           return true
 
+        break
+
+      break
```

Comment:
Shall we remove the break?

Reply:
I believe we'll reach this point when we've already found an item with the desired symbol, but with not enough balance in the desired network, so there's no more need to keep iterating (that's what the 2nd break ensures).

Reply:
Wrote a comment, then deleted it, hope you got the email :D
Short version: I think there's no need to keep iterating when we reach this point, so I'd keep the `break`

Reply:
gotya, that makes sense indeed, thanks for explaining :) that just looked like debugging at first